src/screens/FeedScreen/index.js

```diff
@@ -232,7 +231,7 @@ const FeedScreen = (props) => {
   };
 
   return (
-    <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
+    <View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `SafeAreaProvider` to `View` might cause layout issues on devices with notches or rounded corners. The `SafeAreaProvider` component provides padding to avoid the notch and home indicator on modern iOS devices, as well as safe area insets at the top and bottom of some Android devices. If you're sure that this change won't affect the layout on any device, then it's fine. Otherwise, consider reverting back to using `SafeAreaProvider`.

```diff
- <View>
+ <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->